doc/source/development.rst

```diff
@@ -38,6 +38,31 @@ the following:
 This command is not enough to recompile all C++ unit tests. To do so, see
 `Testing locally`_.
 
+.. _python-develop:
+
+Developing Ray (Python Only)
+----------------------------
+
+.. note:: Unless otherwise stated, directory and file paths are relative to the project root directory.
+
+RLlib, Tune, Autoscaler, and most Python files do not require you to build and compiling Ray. Follow these instructions to develop Ray's Python files locally.
+
+1. Pip install the **latest Ray wheels.** See :ref:`install-nightlies` for instructions.
```

Comment:
Maybe also add a warning here not to re-do another `pip install ray` (followed by installing the wheel via `pip install -U [wheel]`). I think I did that once and it then erases lots of files (weirdly not all) in the git ray folder. One can only get out of this by rolling back the erased files with git to get back to the clean branch.

Reply:
I think this happened after you symlinked; I've added a note below as a warning.